reactive/ovn_chassis_charm_handlers.py

```diff
@@ -91,7 +91,9 @@ def configure_ovs():
         if reactive.is_flag_set('config.changed.enable-dpdk'):
             # Install required packages and/or run update-alternatives
             charm_instance.install()
-        charm_instance.configure_ovs(','.join(ovsdb.db_sb_connection_strs))
+        charm_instance.configure_ovs(
+            ','.join(ovsdb.db_sb_connection_strs),
+            reactive.is_flag_set('config.changed.disable-mlockall'))
```

Comment:
Once the flag is set this will return true every time it is called. If I understand the code correctly it will then restart OVS. This may not be what you intend.

Reply:
Thanks for reviewing this. It seems to work in my testing. For example, when I toggle debug=true/false ovs doesn't get restarted. I think config.changed.disable-mlockall should only be set when disable-mlockall gets changed according to: 
https://charmsreactive.readthedocs.io/en/latest/managed-flags.html

Reply:
ah, I missed the "changed" portion of the flag. You are correct. +1